src/kiutils/schematic.py

```diff
@@ -63,6 +63,9 @@ class Schematic():
     busEntries: List[BusEntry] = field(default_factory=list)
     """The ``busEntries`` token defines a list of bus_entry used in the schematic"""
 
+    busAliases: Dict[BusAlias] = field(default_factory=dict)
```

Comment:
Please use `List[BusAlias] = field(default_factory=list)`. Otherwise we have to keep track of two name attributes (one as dict key and one in BusAlias) that are never in sync.

Reply:
I agree about the out of sync business. Also, I had started with List itself to begin with. 

But, found one issue with that. It is not efficient for lookup by name. The names of buses are present in net labels and `busAliases` being a dictionary makes it efficient to look up for a bus and get its members from it.

If you are insistent on List based design, I will change it to List. Please confirm.

Reply:
Ok, I thought about it and have reverted it to list itself now. Please check.